x-pack/plugins/saved_objects_tagging/common/constants.ts

```diff
@@ -20,4 +20,17 @@ export const tagManagementSectionId = 'tags';
 /**
  * The list of saved object types that are currently supporting tagging.
  */
-export const taggableTypes = ['dashboard', 'visualization', 'map', 'lens', 'search'];
+export const taggableTypes = [
+  'dashboard',
+  'visualization',
+  'map',
+  'lens',
+  'search',
+  // Duplicated types are needed for Fleet integration assets
```

Comment:
@juliaElastic do you think this behavior is expected? or maybe we should use here 
https://github.com/patrykkopycinski/kibana/blob/feat/osquery-so-tagging/x-pack/plugins/fleet/server/services/epm/kibana/assets/tag_assets.ts#L33
`KibanaSavedObjectType` instead? https://github.com/patrykkopycinski/kibana/blob/feat/osquery-so-tagging/x-pack/plugins/fleet/common/types/models/epm.ts#L88

Reply:
[taggableTypes](https://github.com/patrykkopycinski/kibana/blob/feat/osquery-so-tagging/x-pack/plugins/fleet/server/services/epm/kibana/assets/tag_assets.ts#L8) is coming from the `saved-objects-tagging` plugin, @elastic/security-asset-management could you confirm if it is advised to extend this list with `osquery` asset types?
Based on the [README](https://github.com/elastic/kibana/blob/main/x-pack/plugins/saved_objects_tagging/README.md#update-the-taggabletypes-constant-to-add-your-type) this seems to be the recommendation.

Reply:
that's correct, but `KibanaAssetType` from `fleet` plugin is not equal to `KibanaSavedObjects`, first are using `_` in the names while saved objects are named with `-`, so because of that we cannot use `KibanaAssetType` for tagging `osquery*` saved objects

Reply:
@juliaElastic, I have found `KibanaSavedObjectTypeMapping` that I think could be used here to fix this issue
https://github.com/elastic/kibana/pull/143417/files#diff-cb9f22205f0938b97318a003d481d61cf4caa616dbb8c3115f5e60e7436e9efbR34
what do do you think?